requirements.txt

```diff
@@ -3,6 +3,7 @@ pandas >= 0.23.3
 obspy >= 1.1.0
 progressbar2
 xarray >= 0.10.9
+geographiclib
```

Comment:
I am a little reluctant to take this as a dependency, are there any other good options? If not its probably ok.

Reply:
That is fair enough.  Other options might be:
1. Use [`obspy`.geodetics.locations2degrees`](https://docs.obspy.org/master/packages/autogen/obspy.geodetics.base.locations2degrees.html#obspy.geodetics.base.locations2degrees), which gives the degrees we want, but assumes a spherical Earth;
2. Use [`obspy.geodetics.gps2dist_azimuth`](https://docs.obspy.org/master/packages/autogen/obspy.geodetics.base.gps2dist_azimuth.html#obspy.geodetics.base.gps2dist_azimuth), which either uses Obspy's own [vincenty inverse function](https://docs.obspy.org/master/packages/autogen/obspy.geodetics.base.calc_vincenty_inverse.html#obspy.geodetics.base.calc_vincenty_inverse), or `geographiclib` if available. Obspy notes that `geographiclib` is about 4x faster and that their implementation is unstable at antipodes - `geographiclib` shouldn't be. This also return meters, not degrees - `geographiclib` returns both, so we don't have to worry about an additional conversion from meters to degrees.
3. The [`obspy.taup.taup_geo.calc_dist`](https://docs.obspy.org/packages/autogen/obspy.taup.taup_geo.calc_dist.html) probably does everything we want actually, and without adding geographiclib as a hard dependency, and warning about calculating distances on a sphere...  I'm leaning towards using this.